source/extensions/stat_sinks/common/statsd/statsd.cc

```diff
@@ -23,7 +23,7 @@ Writer::Writer(Network::Address::InstanceConstSharedPtr address) {
   fd_ = address->socket(Network::Address::SocketType::Datagram);
   ASSERT(fd_ != -1);
 
-  int rc = address->connect(fd_);
+  const int rc = address->connect(fd_).rc_;
```

Comment:
I'd prefer to keep the SysCallResult instead of an int, so that if the assert fires in a test it's easier in a debugger to see the value of errno.

Reply:
Addressed in 42d0771.